client/src/lib/components/utils/NavItem.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let href: string | null = null;
+</script>
+
+{#if href}
+	<span
+		class="self-center text-xl font-semibold whitespace-nowrap text-dark py-2 px-4 hover:bg-warmGray-300 ease-in-out transition-all duration-200 rounded"
```

Comment:
Trying to move to more utility components like this when there is a repeated, wordy piece of html (in lieu of custom css classes, e.g.)